### PR TITLE
Add transformations for fcqn[action/action-core/canonical] rules

### PR DESCRIPTION
**Pull request by MarkusTeufelberger**
Adds ability to add missing fqcn for actions.

### CODE REVIEW

File: examples/playbooks/fqcn.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- name: FQCN transform test file
+  hosts: localhost
+  tasks:
+    - name: Rewrite shell to ansible.builtin.shell via the fqcn[action-core] transform # noqa: command-instead-of-shell
+      shell: echo This rule should get matched by the fqcn[action-core] rule
+      changed_when: false
+    - name: Rewrite openssh_keypair to community.crypto.openssh_keypair via the fqcn[action] transform
+      openssh_keypair:
```
[the remark this answers is not in the document]
Comment (MarkusTeufelberger) [reply examples/playbooks/fqcn.yml:9]: I mean, I can always just remove these and everything will pass, the only test that actually works without collections available is `fqcn[action-core]` though.

**Comment by ssbarnea** [comment]
@MarkusTeufelberger Ok if I step in to add fixes?

**Comment by MarkusTeufelberger** [comment]
Something that seems a bit problematic still is that newlines get eaten up if an action is followed by an empty line. e.g. like this:

    - name: foo
      include_tasks: foo.yml
    
    - name: bar
      ansible.builtin.apt:
        name:
          - bar

ends up as

    - name: foo
      ansible.builtin.include_tasks: foo.yml
    - name: bar
      ansible.builtin.apt:
        name:
          - bar

**Comment by MarkusTeufelberger** [comment]
Another issue after running this on some private content: Comments get eaten.

Run this against the following content:

    ---
    - name: foo
      hosts: all
      tasks:
        - name: foo
          command: echo foo

        - name: bar
          ansible.builtin.apt:
            name:
              - bar

        - name: baz
          command: echo baz
          # this gets eaten!

**Comment by MarkusTeufelberger** [comment]
I added 2 test cases for this, so it should be easier to verify when it is fixed.

**Comment by ssbarnea** [comment]
Closing this as we will merge #3316 which contains the small fixes to get the CI green. We also observed that release-drafter has a bug related on how it mentions the contributors, https://github.com/release-drafter/release-drafter/issues/1333 but I hope they will fix it soon.

**Pull request closed without merging**